libpod/container_internal.go

```diff
@@ -1120,7 +1120,7 @@ func (c *Container) cleanupRuntime(ctx context.Context) error {
 		return err
 	}
 
-	// If we were Stopped, we are now Exited, as we've removed ourself
+	// If we were Stopped, we are now Exited, as we've removed ourselves
```

Comment:
Although this is not my decision to make, I believe the original usage - implying a singular object - is the correct one. I believe the first-person plural here is the Royal We, which is euphemistically first-person singular. As a reader I find "ourselves" here misleading and confusing; but I will defer to those on the team more familiar with this context.